index.html

```diff
@@ -0,0 +1,177 @@
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-2"></div>
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-3"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-4"></div>
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="about-myself">
+            <div class="myself">
+                <p class="about-title">About <br> Myself</p>
```

Comment:
- [ ] Please note that this is a section title and to be semantically correct you need to use `<hx>` tag (h2 .. h6) instead of `<p> tag` , kindly fix that